Grid/qcd/action/gauge/LLRGaugeAction.h

```diff
@@ -0,0 +1,162 @@
+//
+// Created by dc-bonn2 on 9/17/25.
+//
+
+#include <Grid/qcd/llr_hmc/llr_hmc.h>
+
+#ifndef GRID_TELOS_DEVEL_LLRGAUGEACTION_H
+#define GRID_TELOS_DEVEL_LLRGAUGEACTION_H
```

Comment:
This symbol doesn't need to be namespaces to `TELOS`, or really to `DEVEL`. I'd suggest `QCD_LLR_GAUGE_ACTION_H` to be consistent with the symbol defined in `WilsonGaugeAction.h`.